plans/podman_build_test.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -eox pipefail
+
+rpm -q golang
+
+if [ -f /etc/fedora-release ]; then
+    export TMPDIR=/var/tmp
+fi
```

Comment:
It's unclear to me (a lay person, non-tmt expert) why this is needed.  Perhaps a comment would be helpful to other non-experts?